RecoLocalMuon/CSCRecHitD/src/CSCXonStrip_MatchGatti.cc

```diff
@@ -293,6 +293,7 @@ void CSCXonStrip_MatchGatti::findXOnStrip(const CSCDetId& id,
         noise_level = noise_level_ME1b;
         xt_asymmetry = xt_asymmetry_ME1b;
         const_syst = const_syst_ME1b;
+        [[fallthrough]];
```

Comment:
This should rather ```break```, otherwise ```noise_level``` etc. get overwritten: @ptcox

Reply:
@ptcox, did you have time to look at it? Can you confirm that there is a bug here, and a ```break``` should be present, instead?